packages/@react-types/calendar/src/index.d.ts

```diff
@@ -21,7 +21,7 @@ type MappedDateValue<T> =
   T extends CalendarDate ? CalendarDate :
   never;
 
-export interface CalendarPropsBase {
+export interface CalendarPropsBase extends AriaLabelingProps {
```

Comment:
Is this supposed to have AriaLabelingProps here? It get added when `AriaCalendarProps`/etc gets defined, typically the base props don't extend the aria props

Reply:
ah, might've added it at the wrong level, however, I'm not seeing it as a prop here https://react-spectrum.adobe.com/react-aria/Calendar.html#props and it was throwing an error in the tests trying to use the prop, I'll see about why it might not be exposed up higher